OpenRA.Mods.Common/Activities/Move/VisualMoveIntoTarget.cs

```diff
@@ -48,7 +48,7 @@ public override Activity Tick(Actor self)
 			if (IsCanceled || target.Type == TargetType.Invalid)
 				return NextActivity;
 
-			if (mobile.IsTraitDisabled)
+			if (mobile.IsTraitDisabled || mobile.IsTraitDisabled)
```

Comment:
Both sides of the `||` are the same here... Shouldn't this be `if (mobile.IsTraitDisabled || mobile.IsTraitPaused)`?

Reply:
Fixed.